utils/python/cime_util.py

```diff
@@ -538,8 +535,8 @@ def does_machine_have_batch(machine=None):
     True
     """
     parse_config_machines()
-    if(machine is not None):
```

Comment:
This is another minor nitpick. I'd prefer if a space was left between "if" and the conditional expression. Always using parens in if statements is a personal style, not sure if it's PEP compliant or not.